ui/src/components/schemas/Schemas.tsx

```diff
@@ -43,15 +39,14 @@ export function Schemas() {
       <Divider />
 
       <Space direction="vertical" size="large">
-        {isExplainerShowing && (
-          <Explainer
-            CTA={{ label: "Learn more", url: TUTORIALS_URL }}
-            description="Learn about schema types, attributes, naming conventions, data types and more."
-            handleDismiss={() => setExplainerShowing(false)}
-            image="/images/illustration-explainer.svg"
-            title="Credential schemas explained"
-          />
-        )}
+        <Explainer
+          CTA={{ label: "Learn more", url: TUTORIALS_URL }}
+          description="Learn about schema types, attributes, naming conventions, data types and more."
+          image="/images/illustration-explainer.svg"
+          isDismissable
+          localStorageKey="explainerSchemas"
```

Comment:
I'd consider removing `isDismissable` and typing `localStorageKey` as `string | undefined`.
Like this, we could infer that whenever a `localStorageKey` is present, the explainer is dismissable and it's not otherwise.

The current API forces us to provide a `localStorageKey` even when `isDismissable` is set as false, which makes no sense.

Reply:
Fair point. Fixed.